src/filesystem/File.js

```diff
@@ -31,7 +31,7 @@ define(function (require, exports, module) {
     var FileSystemEntry = require("filesystem/FileSystemEntry"),
         Content         = require("extensions/default/brackets-browser-livedev/nohost/src/content"),
         Path            = require("filesystem/impls/filer/BracketsFiler").Path,
-        defaultHTML     = require("text!filesystem/impls/lib/default.html");
+        defaultHTML     = require("text!extensions/default/brackets-browser-livedev/lib/default.html");
```

Comment:
This is the wrong way around, I think.  We shouldn't have code in `src/` relying on things in an extension.  The reverse is fine, though.  I'd put the default HTML here, and have the extension get it that way.